pose_thumbnails/__init__.py

```diff
@@ -17,12 +17,12 @@
 # ##### END GPL LICENSE BLOCK #####
 
 
-__version__ = '1.0.0'
+__version__ = '1.0.1'
 
 bl_info = {
     'name': 'Pose Library Thumbnails',
-    'author': 'Jasper van Nieuwenhuizen (jasperge) & Sybren A. Stüvel (dr_sybren)',
-    'version': (1, 0, 0),
+    'author': 'Jasper van Nieuwenhuizen (jasperge) & Sybren A. Stüvel (dr_sybren) & Patrick W. Crawford (TheDuckCow)',
```

Comment:
I'd like to shorten the line length. :slightly_smiling_face: 
```suggestion
    'author': ('Jasper van Nieuwenhuizen (jasperge), Sybren A. Stüvel (dr_sybren)'
               ' & Patrick W. Crawford (TheDuckCow)'),
```